pages/api/movies/index.js

```diff
@@ -1,7 +1,13 @@
-import movies from "./movies.json";
+export default async function handler(request, response) {
+  const { search } = request.query;
+  const api_KEY = process.env.tmdbApiKey;
 
-export default function handler(request, response) {
   if (request.method === "GET") {
-    response.json(movies);
+    const result = await fetch(
+      `https://api.themoviedb.org/3/search/movie?query=${search}&api_key=${api_KEY}`
```

Comment:
Use [`URLSearchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) to encode parameters, like:

```js
const result = await fetch(
    `https://api.themoviedb.org/3/search/movie?${new URLSearchParams({ query: search, api_key })}`
);
```

(also rename your `api_KEY` variable just to `api_key`)